### PR TITLE
Add video player doc

### PR DESCRIPTION
**Pull request by ibirman-vbrick**

### CODE REVIEW

**Review by vbrickPMitchell — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -117,6 +117,44 @@ You can remove the webcast from the DOM by calling destroy method.
...
+		const li = document.createElement('li');
+		li.innerHTML = `${new Date().toLocaleTimeString()} ${e}:${stringifyJson(data)}`;
+		logEl.appendChild(li);
+	}));
+  ```
+
+##### Make the player available globally:
+
+```
+globalThis.vbrickEmbed = currentEmbed;
```
Comment (vbrickPMitchell) [inline README.md:149]: That is not something the user needs to do

**Review by vbrickPMitchell — CHANGES_REQUESTED** [review]
Left a few comments

File: README.md [hunk 2]
```diff
@@ -117,6 +117,44 @@ You can remove the webcast from the DOM by calling destroy method.
...
+```
+
+##### Control the player using external commands:
+
+```
+window.vbrickEmbed.play();
+window.vbrickEmbed.pause();
```
Comment (vbrickPMitchell) [inline README.md:156]: dont use `window.`

Reference the same variable defined above, but I would just call it `embed` not `video`

File: README.md [hunk 3]
```diff
@@ -117,6 +117,44 @@ You can remove the webcast from the DOM by calling destroy method.
...
+Use the .on method to watch for events about the video:
+
+```
+['error', 'load', 'videoLoaded', 'volumeChanged', 'captionsChanged', 'seeked', 'playerStatusChanged']
+	.forEach(e => video.on(e, data => {
+		const li = document.createElement('li');
+		li.innerHTML = `${new Date().toLocaleTimeString()} ${e}:${stringifyJson(data)}`;
+		logEl.appendChild(li);
+	}));
+  ```
```
[the remark this answers is not in the document]
Comment (ibirman-vbrick) [reply README.md:144]: ok

File: README.md (hunk 1 above)
Comment (ibirman-vbrick) [reply README.md:149]: ok

File: README.md (hunk 2 above)
Comment (ibirman-vbrick) [reply README.md:156]: ok

File: README.md [hunk 4]
```diff
@@ -117,6 +117,44 @@ You can remove the webcast from the DOM by calling destroy method.
...
 
+#### Embedding a Video
+You can embed a video just like a webcast using an empty div tag.
+
+`<div id="embed"></div>`
+
+Use the embedVideo method to display the video on your web page:
+
+```
+const video = revSdk.embedVideo('#embed', formValues.videoId, {
```
[the remark this answers is not in the document]
Comment (ibirman-vbrick) [reply README.md]: ok

File: README.md (hunk 1 above)
Comment (vbrickPMitchell) [reply README.md:146]: can you remove the section? The user is not necessarily going to be using global variables.

Comment (ibirman-vbrick) [reply README.md]: done

**Review by vbrickPMitchell — APPROVED** [review]

**Pull request merged**